ci/build_python_noarch.sh

```
@@ -21,6 +21,9 @@ PYTHON_CHANNEL=$(rapids-download-from-github "$(rapids-artifact-name conda_pytho
 RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION)
 export RAPIDS_PACKAGE_VERSION
 
+# TODO: Remove before merging. Use rapidsmpf conda packages from rapidsai/rapidsmpf#1081.
+source ./ci/use_conda_packages_from_prs.sh
```

Shellcheck (0.11.0)

[info] Not following: ./ci/use_conda_packages_from_prs.sh was not specified as input (see shellcheck -x).

(SC1091)